RELEASE_NOTES.md

```diff
@@ -2,16 +2,12 @@
 
 ## Summary
 
-<!-- Here goes a general summary of what this release is about -->
+This is a bugfix release that mainly only uses more up to date dependencies and extend the range of supported dependencies.  There is technically one breaking change though, but this is hardly used by anyone.
```

Comment:
I think here should be either `mainly` or `only` but not both

Reply:
Fixed.

Reply:
sorry, my previous comment about `s/extend/extends` was missed. do you want to update it? Anyway I'm happy  to approve it as it is